DigitalLearningSolutions.Data/Models/CustomPrompts/CentreCustomPromptsResult.cs

```diff
@@ -0,0 +1,37 @@
+﻿namespace DigitalLearningSolutions.Data.Models.CustomPrompts
+{
+    public class CentreCustomPromptsResult
+    {
+        public int CentreId { get; set; }
+
+        public int CustomField1PromptId { get; set; }
+        public string? CustomField1Prompt { get; set; }
+        public string? CustomField1Options { get; set; }
+        public bool CustomField1Mandatory { get; set; }
+
+        public int CustomField2PromptId { get; set; }
```

Comment:
Do we need these ids in the db level or is prompt name sufficient (as we only use the id for join)?

Reply:
I'm using it to determine if it is a prompt or not in the CustomPromptService, if promptId != 0 && prompt != null then it is a prompt.

Reply:
Would prompt != null be sufficient?

Reply:
Having looked at the database, it probably should be. Will refactor it to remove the IDs